### DISCUSSION TITLE
Step by Step guide for non expert to Install Slinger for Multiple 500s - Windows

### DISCUSSION THREAD

**Discussion by tryin100** [GerryDazoo/Slinger#413] [General]
Here's an updated pdf Version 3.0 where I detailed how to install Slinger, configure, and run it.

It's written for Windows users, slingbox 500s, and using VLC media player, but it could also be useful for other Windows users who, like me, aren't as talented as the other posters here on Slinger.

[Simple step by step Slinger installation for multiple 500s v03.pdf](https://github.com/GerryDazoo/Slinger/files/10156914/Simple.step.by.step.Slinger.installation.for.multiple.500s.v03.pdf)

**Comment by floormanager1** [comment]
Great work!

**Reply by sp222222** [reply]
page 17 about port blocking needs to be waaaay further at the top if not top. but nicely done there.

**Comment by womeara6** [comment]
Hi Guys.  I tried to test the parental controls blocking technique using my laptop and they didn't work.  I selected the my machine and blocked the sling.com website, rebooted router and laptop and tried the url.  Unfortunately, it took me right to the site.  So much for blocking porn sites from my grandkids.  We need a better blocking solution, I think.

Thanks,
Bill

**Reply by sp222222** [reply]
Bill, block port 5678 both outbound and inbound. there is nothing else to do except that to prevent bricking

**Reply by tryin100** [reply]
If I find that my 3 slingboxes are actually using different ports than 5678 should I block THOSE ports - or, is 5678 a port that every slingbox uses to communicate with the company systems?
thanks

**Reply by GerryDazoo** [reply]
5678 has nothing to do with the port numbers the slingboxes use. 5678 is a port number on one the the Sling servers.

**Comment by womeara6** [comment]
Hi Gerry.  I tested the parental controls on the Linksys WRT1900AC as per a prior suggestion and they don’t appear to work. I blocked my laptop from accessing sling.com and it didn’t work after rebooting the router and laptop.  While I blocked the 500 from the internet access I can’t be sure that it is working either.  I intend to contact Linksys support tomorrow on this parental control issue.

 

So, since I am at a loss on how to block that port, I just pulled the 500 completely off the network and shut it off.  I’ll need to find out how to block ports before restarting the 500.

 

Thanks again.

 

Bill

**Comment by GerryDazoo** [comment]
OK. I have a few comments on the doc.
1) "The IP addresses are 192.168.something", 
This is common but it could be any of the non-routavle IP address blocks. It depends on your you local LAN is configured on your router. 192.168.x.x, 172.16.0.0-172.16.31.255 or 10.x.x.x

2)In the config.ini "Code=InputHereTheSameNumberasYouEnteredforVideoSourceAbove"
You don't need this. It's only required if you want to use the remote config for a different videosource on your slingbox. This is very uncommon.

3) "THE
PORT NUMBER TO FORWARD IS WHATEVER YOU CHOSE TO USE ABOVE FOR THE SERVER PORT WHEN
CREATING YOUR CONFIG.INI FILE!
Enter that port number under both ‘internal’ and ‘external’ port. For protocol choose ‘both’ and enter the
internal IP address of the computer where you are running the server that you got in Step 4."

The external port number can be anything you want. It does nor have to match the internal port number.
The protocol to use is TCP. Allow UDP does no good.

Section 12.
These are all issues with using VLC. Don't do that.
a) On VLC turn on "loop 1". It will auto reconnect when the stream drops. This is an issue with VLC not being able to dynamically resync the video stream when the SB input changes.
b) More VLC stuff. All the video/audio are sent in the same TCP connection. If your video and audio get out of sync it's a problem with your player.

Section 13: Putting this in your startup folder will only get it to run when you log in. If you actually want it to run on boot up, you need to run it as a windows service.


4) Section 8. A note on using a DYNDNS service might be helpful, Also the previous text about setting up the external port map should go here. It's only needed if you're not watching from "home". People will waste a lot of time setting this up even if they don't need it.

**Reply by tryin100** [reply]
i've updated with your suggestions.
re: code= .   thanks. I didn't know it's only required if you want to use the remote config for a different videosource on your slingbox.  But since it does no harm I'll leave it for simplicity.
re: including a note on using a DYDNS service.  I'd love to.  I'd love to use one myself.  I just don't have the skills to write up a simple one.

**Reply by GerryDazoo** [reply]
I don't think you need to go into detail. The DYNDNS providers have documentation. Maybe just a single line example'
http://YourDYNDNSdomainname:YourExternalPortNumber/YourURLbase/YourSlingBoxName

**Comment by tryin100** [comment]
thanks again. having real experts i can learn from is fantastic.

**Reply by GerryDazoo** [reply]
Another note: If you only have one slingbox on your LAN you don't need to put the IP address and port in the config.ini. the server code will find it and use it. All you really need is the password and the sbtype if it's not a 500/350/M1.
PS. The next version of the server code will also detect the slingbox model so you won't even need the sbtype. I'm also considering having the code automatically try the "admin" password on a login failure so after a factory reset (Don't do it)  you can turn up the server without needing to configure any of that info,

**Comment by ChrizK-oldGit** [comment]
> Here's an updated pdf Version 3.0 where I detailed how to install Slinger, configure, and run it.
> 
> It's written for Windows users, slingbox 500s, and using VLC media player, but it could also be useful for other Windows users who, like me, aren't as talented as the other posters here on Slinger.
> 
> [Simple step by step Slinger installation for multiple 500s v03.pdf](https://github.com/GerryDazoo/Slinger/files/10156914/Simple.step.by.step.Slinger.installation.for.multiple.500s.v03.pdf)

Just a quick comment about the title of the discussion.  Having a quick look, I think the instructions are for windows users using **slingbox_server.exe**, not 'installing slinger' as I hoped (IMO installing implies using slinbox_server.py, having to install python and modules ...which I am having problems with, hence looking).

That all said, maybe I will drop the idea of using slingbox_server.py and just go with the .exe, now that I have come across your very nicely written instructions :)

**Reply by tryin100** [reply]
I'm happy it's helpful.  I am, admittedly, a technology amateur relative to others here.  You are correct about my word choice.

**Reply by mike-uk** [reply]
Thank you for making it pop back up to the top. Eventhought I installed on linux, I learn few things from this document. The loop one in vlc is a good one for device that can't use slingerplayer. It was driving me crazy on every time I change channel it stop. Thanks tryin100 for making this.
